### PR TITLE
Added hello world h1

### PR DESCRIPTION
**Pull request by StevenRafferty**

### CODE REVIEW

**Review by sol3uk — APPROVED** [review]
lgtm!

File: index.html [hunk 1]
```diff
@@ -5,5 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <h1>Hello World</h1>
```
Comment (sol3uk) [inline index.html:9]: This is a bit overused!

Comment (sol3uk) [reply index.html:9]: @StevenRafferty

**Review by sol3uk — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (StevenRafferty) [reply index.html:9]: Ok thanks for the feedback

**Pull request merged**